_posts/2025-02-01-lts-cadence.adoc

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: 'Quarkus LTS - Release Cadence Explained'
+date: 2025-02-01
+tags: release
+synopsis: 'We are introducing a new release cadence for the LTS streams.'
+author: cescoffier
+---
+:imagesdir: /assets/images/posts/lts-cadence
+
+
+Quarkus releases an https://quarkus.io/blog/lts-releases/[LTS] (Long-Term Support) version every six months.
+LTS is designed for users who prioritize stability over new features.
+These versions are maintained for one year and receive critical bug and CVE fixes.
+An overlap period allows a smooth upgrade to the next LTS.
+
+Until now, LTS micro-releases (e.g., 3.8.1, and 3.8.2) have occurred regularly but without a predictable schedule.
+**We’re changing this.**
+
+****
+**TLDR:**
+LTS releases will follow a predictable cadence, with micro-releases every two months.
+The <<new-cadence>> section provides more details.
+****
+
+== Releases, Releases, and More Releases
+Since its inception, Quarkus has followed a fast-paced release cycle:
+* _Minor releases:_ Once per month (e.g., 3.16, 3.17).
+* _Micro-releases:_ Weekly (e.g., 3.17.1, 3.17.2).
```

Comment:
This isn't rendering as a list:

<img width="831" alt="image" src="https://github.com/user-attachments/assets/60e713ca-2811-4d51-9782-7b83e827b78f" />

Reply:
ah thanks! Forgot a new line.